index.js

```diff
@@ -309,6 +309,25 @@ Assert.add('a, an', function typecheck(of, msg) {
 });
 
 /**
+ * Asserts if the given value is the correct type from a list of types.
+ * The same caveats regarding `typeof` apply as described in `a`, `an`.
+ * 
+ * @param {String[]} ofs Acceptable types to check against
+ * @param {String} msg Reason of failure.
+ * @returns {Assert}
+ * @api public
+ */
+Assert.add('eitherOfType, oneOfType', function multitypecheck(ofs, msg) {
+  var value = type(this.value)
+    , expect = format('`%j` (%s) to @ be a %s', this.value, value, ofs.join(' or a '));
+
+  return this.test(ofs.some(function(of) {
```

Comment:
Would be nice if we did this without `.some` as this library works in IE6 as well.